packages/cli/src/commands/setup/deploy/providers/aws-serverless.js

```diff
@@ -24,19 +25,26 @@ custom:
 
 provider:
   name: aws
-  runtime: nodejs12.x
+  runtime: nodejs14.x
   region: us-east-2 # This is the AWS region where the service will be deployed.
   httpApi: # HTTP API is used by default. To learn about the available options in API Gateway, see https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-vs-rest.html
     cors: true
```

Comment:
`cors: true` is serverless shorthand for wild card origin and a bunch of headers, the biggest problem is the wildcard as that it's not allow by browsers for requests with credentials. Here's my suggestion for what we should add instead.

```suggestion
    cors:
      allowedOrigins:
        - '*' # the * wildcard will need be replaced with your domain to allow requests with credentials.
      allowCredentials: true
      allowedHeaders:
        - authorization
        - auth-provider
        - content-Type
```

Though we could also add these common aws headers that could be useful depending on what other aws services a Redwood user might employ.
```
              - X-Amz-Date
              - X-Api-Key
              - X-Amz-Security-Token
              - X-Amz-User-Agent
```

Reply:
I'm not sure, but that sounds good if there is an option like that.